service/src/main/java/com/ge/predix/acs/audit/PredixEventMapper.java

```diff
@@ -36,11 +39,18 @@ public AuditEventV2 map(final AuditEvent auditEvent) {
         try {
             payload = OBJECT_MAPPER.writeValueAsString(auditPayload);
         } catch (JsonProcessingException e) {
-            LOGGER.warn("Unable to convert audit payload to json: " + auditPayload.toString());
+            LOGGER.warn("Unable to convert audit payload to json: {}" + auditPayload, e);
         }
 
+        String correlationId = auditEvent.getCorrelationId();
```

Comment:
please add detailed comments here for this block which is setting value of correlationId